llabake/Hello-Books
# bug(user object): update user object in component

#### What does this PR do?
* This PR adds updates the user object on favorite and profile page

#### Description of Task to be completed?
* remove trailing slash from user's borrowed book url
* update user object on profile page
* update user object on favorite page
* fix broken profile page url for user's borrowed books
* paginate favorite page in controller

#### How should this be manually tested?
* clone the repo
* cd into the project directory
* run npm install to install all the dependency
* run npm run start-dev to start the back-end server
* run npm run dev to start the front-end server
* sign in  to the application
* then navigate to the favorite and profile page it should load your user detail

#### What are the relevant pivotal tracker stories?
157527617

#### Any background context you want to add?
N/A

#### Screenshots
N/A
